src/status.cpp

```diff
@@ -271,6 +272,7 @@ const std::unordered_map<const StatusCode, grpc::StatusCode> Status::grpcStatusM
     {StatusCode::MODEL_SPEC_MISSING, grpc::StatusCode::INVALID_ARGUMENT},
     {StatusCode::INVALID_SIGNATURE_DEF, grpc::StatusCode::INVALID_ARGUMENT},
     {StatusCode::PIPELINE_DEMULTIPLEXER_NO_RESULTS, grpc::StatusCode::ABORTED},
+    {StatusCode::ALLOW_CACHE_WITH_CUSTOM_LOADER, grpc::StatusCode::INVALID_ARGUMENT},
```

Comment:
Why is this translated to http/grpc? This status is never going to be exposed over the network, right? Is this only internal status we see in logs?

Reply:
It seems like it is only status that we see in logs. I will remove the mappings.